server/controllers/erasure-search.js

```diff
@@ -0,0 +1,97 @@
+const { default: log } = require('@financial-times/n-logger');
+const operation = 'erasure-search';
+
+async function searchUserTableByUuid (db, uuid, requestedBy) {
+	try {
+		const userTableResults = await db.run(`SELECT * FROM syndication.users WHERE user_id = '${uuid}';`);
```

Comment:
It looks like this could allow SQL injection. I don't know the Postgres `massive` library so it's hard to tell; looking at the docs it says the `run` method is deprecated and doesn't provide much info about guarding against injection.

Reply:
This is currently using v3.1.0 of `massive`, and the example provided in [the docs for that version](https://gitlab.com/dmfay/massive-js/tree/v3.1.0#raw-sql) shows how to provide parameters to your query. Parameterised Queries are how Postgres protects against SQL injection.

I'll update the queries here to use parameters instead of string literals.

Reply:
Just an FYI #163 upgrades massive